Dockerfile

```diff
@@ -69,6 +69,9 @@ RUN curl -L ${NEXUS_DOWNLOAD_URL} --output nexus-${NEXUS_VERSION}-unix.tar.gz \
     && mv ${SONATYPE_WORK}/nexus3 ${NEXUS_DATA} \
     && ln -s ${NEXUS_DATA} ${SONATYPE_WORK}/nexus3
 
+# Remove java settings from nexus.vmoptions
```

Comment:
an improved comment would be stating WHY - the WHAT is mostly obvious from reading the code.

Reply:
done , thx for pointing it